tests/e2e/api/controllers/_changes.spec.js

```diff
@@ -4,7 +4,8 @@ const _ = require('underscore'),
       http = require('http'),
       querystring = require('querystring'),
       constants = require('../../../constants'),
-      auth = require('../../../auth')();
+      auth = require('../../../auth')(),
+      semver = require('semver');
```

Comment:
I can't find where `semver` is listed in a package.json so I think this is just a transitive dependency currently so we can't rely on it being there forever. Add it to the appropriate package.json so it doesn't disappear!